security/run.sh

```diff
@@ -9,4 +9,8 @@ echo "Deleting project"
 curl -X POST -u $SQ_TOKEN: $SQ_URL/api/projects/delete?project=$PK
 
 echo "Running analysis"
-mvn clean verify sonar:sonar -Dsonar.host.url=$SQ_URL -Dsonar.login=$SQ_TOKEN -Dsonar.projectKey=$PK
+mvn clean verify sonar:sonar -Dsonar.host.url=$SQ_URL -Dsonar.login=$SQ_TOKEN -Dsonar.projectKey=$PK \
+   -Dsonar.scm.disabled=true -X \
+   -Dsonar.security.sources.javasecurity.S3649=s3649JavaSqlInjectionConfig.json \
+   -Dsonar.security.sanitizers.javasecurity.S3649=s3649JavaSqlInjectionConfig.json \
+   -Dsonar.security.sinks.javasecurity.S3649=s3649JavaSqlInjectionConfig.json
```

Comment:
As I said in my general comment, I find it confusing here that we're passing a file containing multiple config types as the value for each separate config type. But I see upon reviewing our docs that this is the only way to do it, and indeed for this example keeping all the config in one file seems more concise.